src/python/WMCore/Services/pycurl_manager.py

```diff
@@ -418,11 +413,11 @@ def getdata(urls, ckey, cert, headers=None, options=None, num_conn=100, cookie=N
                 curl.setopt(pycurl.COOKIEFILE, cookie[url])
                 curl.setopt(pycurl.COOKIEJAR, cookie[url])
             if sys.version.startswith('3.'):
-                bbuf = io.BytesIO()
-                hbuf = io.BytesIO()
+                bbuf = BytesIO()
+                hbuf = BytesIO()
             else:
-                bbuf = StringIO.StringIO()
-                hbuf = StringIO.StringIO()
+                bbuf = StringIO()
+                hbuf = StringIO()
```

Comment:
Shouldn't this just be removed? Attempting to replace `StringIO.StringIO()` with `io.StringIO` would fail because the former can handle bytes and strings, while the second can't. 
If we have `io` in python2, then `io.BytesIO` should work for both python2 and python3 with pycurl.

Reply:
You're correct Kenyi. It does fix the unit test I was looking at btw, let's see how it goes with the others.